src/BizHawk.Client.Common/movie/import/LsmvImport.cs

```diff
@@ -150,7 +150,6 @@ protected override void RunImport()
 						case "sgb_ntsc":
 						case "sgb_pal":
 							platform = VSystemID.Raw.SNES;
```

Comment:
This isn't your fault, but this statement should be changed to `platform = VSystemID.Raw.SGB`.